frontend/public/components/utils/index.tsx

```diff
@@ -59,3 +59,4 @@ export { default } from './operator-backed-owner-references';
 export * from './field-level-help';
 export * from './details-item';
 export * from './types';
+export * from './use-url-hook';
```

Comment:
lets not add this hook to index file. We want to remove all indexes as they cause issues due to circular deps

Reply:
fixed